tests/auth.setup.ts

```
@@ -36,5 +40,21 @@ setup("Authentication as a PRO user", async ({ request }) => {
     accessToken,
     "Access token should be defined in response"
   ).toBeDefined();
-  process.env["ACCESS_TOKEN"] = accessToken;
+  return accessToken;
+}
+
+setup("Authentication as a PRO user", async ({ request }) => {
+  process.env["ACCESS_TOKEN"] = await getAccessToken(
+    process.env.EMAIL_PRO!,
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)